src/components/plots/helpers/populateHeatmapData.js

```diff
@@ -238,7 +260,7 @@ const populateHeatmapData = (
 
   data.trackColorData = trackData.map((datum) => datum.trackColorData).flat();
   data.trackGroupData = trackData.map((datum) => datum.groupData).flat();
-
+  data.clusterSeparationLines = trackData[0].clusterSeparationLines;
```

Comment:
is it possible that `trackData` will be empty? Because then `trackData[0]` will error out

Reply:
no, it will return something from generateTrackData